source/gui/addonStoreGui/controls/storeDialog.py

```diff
@@ -370,6 +387,12 @@ def onListTabPageChange(self, evt: wx.EVT_CHOICE):
 		if not self.addonListTabs.HasFocus():
 			self.addonListTabs.SetFocus()
 
+	def onColumnFilterChange(self, evt: wx.EVT_CHOICE):
+		colIndex = evt.GetSelection()
```

Comment:
shouldn't this just call `setSortField`?

Reply:
Yes, done.